### PR TITLE
Add deploy SecretFlow in WSL(Windows Subsystem for Linux) with simulation mode

### PR DESCRIPTION
**Pull request by Mingbo-Lee**
I add document guide to deploy SecretFlow in WSL(Windows Subsystem for Linux) with simulation mode

### CODE REVIEW

File: docs/getting_started/deployment.md [hunk 1]
```diff
@@ -267,6 +267,286 @@ spu = sf.SPU(cluster_def=cluster_def)
 
 Due to the complexity of the Ray communication protocol, including cross-node mutual access and using many ports, it's really complex and error-prone to deploy multiple containers on multiple machines using the bridge network to build a Ray cluster for simulation. Ray's official support for this is not good, so we do not recommend this deployment mode.
 
+### Deploy SecretFlow in WSL(Windows Subsystem for Linux) with simulation mode
+
+If you use SecretFlow in WSL2, you could deploy SecretFlow with different distributions in WSL2.
```
Comment (zhouaihui) [inline docs/getting_started/deployment.md:272]: Why is there emphasis on two different Linux distribution packages here? WSL does not allow run the same distribution more than once at the same time?

File: docs/getting_started/deployment.md [hunk 2]
```diff
@@ -267,6 +267,286 @@ spu = sf.SPU(cluster_def=cluster_def)
...
+
+   - Install SecretFlow
+
+        ```bash
+        pip install -U secretflow
+        ```
+    
+3. Get the IP address of your WSL.
+
+Due to the implementation principle of WSL, different distributions in WSL share the same IP address, then we only could use it to deploy SecretFlow just in a single machine mode. Use the command`ifconfig`to get the IP address of your WSL.
```
Comment (zhouaihui) [inline docs/getting_started/deployment.md:348]: command`ifconfig`to:  Add missing blank.

File: docs/getting_started/deployment.md [hunk 3]
```diff
@@ -267,6 +267,286 @@ spu = sf.SPU(cluster_def=cluster_def)
...
+if `command not found`appears, you could install `net-tools`, you could run the following command to install it in Ubuntu.
```
Comment (zhouaihui) [inline docs/getting_started/deployment.md:349]: Add missing blank: if `command not found`appears

**Comment by zhouaihui** [comment]
Nice Work! Can you update the CN po file?

File: docs/getting_started/deployment.md (hunk 1 above)
Comment (Mingbo-Lee) [reply docs/getting_started/deployment.md:272]: In Microsoft Store, if you have install one Linux distribution package such as Ubuntu 22.04.2 LTS, you could not  install another independent Ubuntu 22.04.2 LTS, which means that you run the same distribution more than once at the same time is equal to that you open more than one terminal based on the same distribution.In this case, I emphasis on two different Linux distribution packages here to simulate two machine host which have indenpendent file system and indenpendent users but same IP address and then we deploy SecretFlow in WSL with simulation mode but not single machine mode.

Comment (zhouaihui) [reply docs/getting_started/deployment.md:272]: I C.

Comment (Mingbo-Lee) [reply docs/getting_started/deployment.md:272]: Strictly speaking, Maybe we can use migration,import and rename of wsl to realize the deploying SecretFlow in WSL with same distribution,  but it maybe not suitable for guide document for the reason that it bring extra complex operation and other issue.
In this case, I recommend two different Linux distribution.
migration,import and rename of wsl ref to: 
https://blog.csdn.net/weixin_43408232/article/details/128606034?spm=1001.2014.3001.5501
here mention how to migrate wsl.

File: docs/locales/zh_CN/LC_MESSAGES/getting_started/deployment.po [hunk 4]
```diff
@@ -429,18 +431,240 @@ msgid ""
 "for this is not good, so we do not recommend this deployment mode."
 msgstr "如果您需要在多台机器上使用bridge网络部署多个容器来搭建Ray集群，由于Ray的通信协议复杂(使用到很多端口且会发生跨节点互相访问），导致需要每台机器上都要透出很多端口，操作很复杂且很容易出错。Ray官方对此的支持也并不好，所以我们非常不推荐这种部署模式。"
 
-#: ../../getting_started/DEPLOYMENT.md:270
+#: ../../getting_started/deployment.md:270
+#, fuzzy
```
Comment (zhouaihui) [inline docs/locales/zh_CN/LC_MESSAGES/getting_started/deployment.po:435]: Remove all `fuzzy`. CN  with `fuzzy` won't take effect. Same below

**Review by zhouaihui — DISMISSED** [review]
LGTM

**Review by zhouaihui — APPROVED** [review]

**Pull request merged**